packages/vaadin-icons/assets/vaadin-font-icons.json

```diff
@@ -7714,5 +7714,70 @@
       "cash",
       "coins"
     ]
+  },
+
+
+
+  {
+    "name": "funcion",
+    "code": "e64c",
+    "categories": [
+      "Web application"
+    ],
+    "meta": [
+      "math",
+      "code"
+    ]
+  },
+  {
+    "name": "megafone",
+    "code": "e6b4",
+    "categories": [
+      "Social",
+      "Item"
+    ],
+    "meta": [
+      "campaign",
+      "marketing"
+    ]
+  },
+  {
+    "name": "palete",
+    "code": "e667",
+    "categories": [
+      "Tools",
+      "Item"
+    ],
+    "meta": [
+      "color",
+      "paint"
+    ]
+  },
+  {
+    "name": "buss",
+    "code": "e90c",
```

Comment:
Is the code property used somewhere when pointing an icon? Or is it rather meta information as well as the category property?

Reply:
If the icons were used as a font icon, I guess the codes would be needed. But currently, that's not supported as we only publish "iconset.js", "vaadin-icons.js", "vaadin-iconset.js" from this package.